packages/fx-core/src/common/template-utils/templatesActions.ts

```diff
@@ -227,4 +246,11 @@ export async function scaffoldFromTemplates(
       await context.onActionError(action, context, e);
```

Comment:
Send error event here?

Reply:
In short, I think whether we need to sendErrorTelemetry here depends on what kind of BI insights do we want to get from it. 
1. It's kind of tricky because of the design - I mean, if there's some error throwed here, `context.onActionError` may not treat it as an Error (Fallback logic). 
2. I choose not to `sendErrorTelemetry` here - Because of the fallback logic, errors are in low possibility, and we have Error Distribution + Error Details Table for each plugin.